contents/terms.mdx

```diff
@@ -106,6 +108,8 @@ By signing up for PostHog Scale License, you and any entity that you represent (
 
 2.3 Customer will be responsible for maintaining the security of Customer’s account, passwords (including but not limited to administrative and User passwords) and files, and for all uses of Customer account with or without Customer’s knowledge or consent.
 
+2.4 Customer will not air gap their PostHog Scale instance without prior permission from PostHog, as this prevents us from being able to bill you correctly based on usage. Flat usage billing is only available under the PostHog Enterprise License.
```

Comment:
Can we extend this to say only for Annual up-front commitments?

Reply:
We can but is it strictly necessary to say that? Given we already have discretion to offer this, and saying it's only for annual up-front might reduce your flexibility.

Reply:
We can leave it out - I'm thinking of avoiding another Suprdaily situation but maybe we just do that on an ad hoc basis if we feel it's appropriate